Assets/Art/Enemies/FlappyBatController.controller

```diff
@@ -0,0 +1,72 @@
+%YAML 1.1
```

Comment:
I'd separate out the animation assets into their own Animation/Enemies folder.  Once we have a few art assets things will get cluttered real fast.